api/npm.ts

```diff
@@ -80,7 +88,13 @@ async function pkgJson (pkg: string, tag = 'latest'): Promise<any> {
 }
 
 async function info (topic: string, pkg: string, tag = 'latest') {
-  const meta = await (tag === 'latest' && pkg[0] !== '@' ? npmMetadata(pkg) : pkgJson(pkg, tag))
+  // ver === 'latest', non-scoped package use npmMetadata (npm), all others use unpkg
+  // optionally disable unpkg to request all info from NPM
+  const meta = await(
+    process.env.DISABLE_UNPKG === 'true' || (tag === "latest" && pkg[0] !== "@")
```

Comment:
`DISABLE_UNPKG` seems very specific, if it's not a must-have, I prefer not to have it

Maybe we could find a better solution for this later, or maybe just wait better npm registry api to erase this problem.

Reply:
Alternatively I could disable unpkg usage only when a custom npm registry is defined? 

I kind of need to disable it as the primary usage for self hosting is for referencing internal packages. Packages that unpkg won't know about.